conf/config.go

```diff
@@ -18,6 +18,7 @@ const (
 	HerokuIntegrationSalesforceAuthPath       = "/invocations/authentication"
 	HerokuIntegrationDataActionTargetAuthPath = "/data_action_targets/authenticate"
 	YamlFileName                              = "heroku-integration-service-mesh.yaml"
+	AddonAuthUrlFormat                        = "https://applink.heroku.com/addons/%s/connections/salesforce"
```

Comment:
@cwallsfdc I believe we can omit the prefix and cover staging and prod urls, something like this: ```heroku.com/addons/%s/connections/salesforce```